macros/vendored/dbt_core/format_column.sql

```diff
@@ -1,5 +1,21 @@
 {% macro format_column(column) -%}
+  {{ return(adapter.dispatch('format_column', 'codegen')(column)) }}
+{%- endmacro %}
+
+{# Vendored from: https://github.com/dbt-labs/dbt-adapters/blob/c7b12aee533184bad391a657d1753539d1dd496a/dbt/include/global_project/macros/relations/column/columns_spec_ddl.sql#L85-L89 #}
+{% macro default__format_column(column) -%}
   {% set data_type = column.dtype %}
   {% set formatted = column.column.lower() ~ " " ~ data_type %}
   {{ return({'name': column.name, 'data_type': data_type, 'formatted': formatted}) }}
 {%- endmacro -%}
+
+{# Vendored from: https://github.com/dbt-labs/dbt-bigquery/blob/4d255b2f854d21d5d8871bdaa8d7ab47e7e863a3/dbt/include/bigquery/macros/utils/get_columns_spec_ddl.sql#L1-L5 #}
+{# But modified to handle https://github.com/dbt-labs/dbt-codegen/issues/190 #}
+{% macro bigquery__format_column(column) -%}
+  {% set data_type = column.data_type %}
+  {% if column.mode.lower() == "repeated" and column.dtype.lower() == "record" %}
+    {% set data_type = "array" %}
+  {% endif %}
+  {% set formatted = column.column.lower() ~ " " ~ data_type %}
+  {{ return({'name': column.name, 'data_type': data_type, 'formatted': formatted}) }}
+{%- endmacro -%}
```

Comment:
Adding trailing newline:

```suggestion
{%- endmacro -%}

```

This will remove this red icon from the diff:

<img width="189" alt="image" src="https://github.com/user-attachments/assets/1dbdfdf8-a4f1-4613-ae2f-0c936591c889" />